### PR TITLE
MOB-194-Configure air Quality Messages

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
#### Summary of Changes (What does this PR do?)

- Configure Air Quality messages on Insights Page

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [MOB-194](https://airqoteam.atlassian.net/browse/MOB-194) 

#### Screenshots (optional)

[MOB-194]: https://airqoteam.atlassian.net/browse/MOB-194?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mobile/lib/models/insight.dart [hunk 1]
```diff
@@ -22,43 +22,70 @@ class Insight with EquatableMixin {
...
+            'The air quality $dateAdverb in ${airQualityReading.name} $verb ${airQuality.title} and can cause a health emergency.';
+        break;
+    }
       if (forecastValue != null) {
-        forecastMessage =
-            'Expect today\'s conditions to be ${Pollutant.pm2_5.airQuality(forecastValue).title.toLowerCase()}';
+      switch (Pollutant.pm2_5.airQuality(forecastValue)) {
+        case AirQuality.good:
+        case AirQuality.moderate:
+        case AirQuality.hazardous:
```
Comment (NoahNsimbe) [inline mobile/lib/models/insight.dart:67]: Thanks @BenjaminSsempala

File: mobile/lib/models/insight.dart [hunk 2]
```diff
@@ -22,43 +22,70 @@ class Insight with EquatableMixin {
...
-    if (airQualityReading.dateTime.isYesterday()) {
-      airQualityMessage =
-          'The average air quality yesterday in ${airQualityReading.name} was ${airQuality.title}.';
-    } else if (airQualityReading.dateTime.isAPastDate()) {
-      airQualityMessage =
-          'The average air quality in ${airQualityReading.name} was ${airQuality.title}.';
-    } else if (airQualityReading.dateTime.isToday()) {
-      airQualityMessage =
-          'The average air quality in ${airQualityReading.name} is currently ${airQuality.title}.';
+    switch (airQuality.title) {
```
Comment (NoahNsimbe) [inline mobile/lib/models/insight.dart:37]: Thanks @BenjaminSsempala . You can Switch airQaulity. For exmaple what you did here https://github.com/airqo-platform/AirQo-frontend/pull/1107#discussion_r1140451489

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Comment by NoahNsimbe** [comment]
@BenjaminSsempala @Baalmart could we make these messages dynamic. Something like fetching them from an endpoint??

File: mobile/lib/models/insight.dart (hunk 2 above)
Comment (BenjaminSsempala) [reply mobile/lib/models/insight.dart]: Thanks @NoahNsimbe , Done

**Comment by BenjaminSsempala** [comment]
> @BenjaminSsempala @Baalmart could we make these messages dynamic. Something like fetching them from an endpoint??

@NoahNsimbe Yes, I've added this as a task to the backlog

**Pull request merged**